### PR TITLE
Add `test.only` helper to test one case

### PR DESCRIPTION
**Pull request by fisker**
Can be used as

```js
test({
  valid: [
    test.only`code`,
    test.only('code'),
    test.only({code: 'code'}),
    {code: 'code', only: true},
  ],
})
```

### CODE REVIEW

File: scripts/internal-rules/no-test-only.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+				}
+
+				context.report(problem);
+			},
+		};
+	},
+	meta: {
+		fixable: 'code',
+		messages: {
+			[messageId]: '`test.only` cannot be used.',
```
Comment (sindresorhus) [inline scripts/internal-rules/no-test-only.js:71]: Not immediately clear from the message why it cannot be used?

Comment (fisker) [inline scripts/internal-rules/no-test-only.js:71]:
```suggestion
			[messageId]: '`test.only` should only be used for debugging purposes, please remove it before commit.',
```

Better?

**Comment by sindresorhus** [comment]
@fisker Do you plan any more changes this week? If not, I'm going to do a new release.

**Comment by fisker** [comment]
Unfortunately, I have very limited time for open source projects this year. Don't have plan for other changes now, you can release if you want.

**Pull request merged**